composer.json

```diff
@@ -10,7 +10,7 @@
     "require": {
         "ezsystems/platform-ui-assets-bundle": "^3.0.0",
         "ezsystems/repository-forms": "^1.2",
-        "ezsystems/ezpublish-kernel": "^6.6",
+        "ezsystems/ezpublish-kernel": "^6.6.7",
```

Comment:
Nope, or it's like forcing an update to 1.7, is it not ?

Anyway, according to Damien, the change is backward compatible here as well, meaning no requirement changes.